File: benchmarks/python3/instantiation.py
# This benchmark stresses instance creation and initializer calling.


class Foo:
    def __init__(self):
        pass


i = 0
while i < 500000:
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    Foo()
    i = i + 1

File: benchmarks/nodejs/instantiation.js
// this benchmark stresses instance creation and constructorializer calling.

class Foo {
    constructor() {}
}

var i = 0;
while (i < 500000) {
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    new Foo();
    i = i + 1;
}
